app/views/moderations/actions_panel.html.erb

```diff
@@ -101,7 +101,7 @@
       </button>
 
       <div id="adjust-tags-options" class="adjust-tags-options dropdown-options hidden">
-        <% if current_user.any_admin? %>
+        <% if policy(@moderatable).elevated_user? %>
```

Comment:
This is logically the same as `current_user.elevated_user?`; my preference is that we want to say "What is the action they are taking?"

I think you're alluding to that in the `should_show_adjust_tags` method.

Would `policy(@moderatable).allow_tag_adjustment?` make sense?  Do we have another method instead that does this logic?

Reply:
I think so; the reason is falls back to the philosophical question of needing to eventually ask "What all articles can have their tags adjusted?"  (Which is not this question but is something I would expect a policy to answer)

I would also look at this new method calling the old method AND doing the count check.